### ISSUE TITLE
Group message responses seem to not send?

### ISSUE THREAD

**Issue by j6k4m8** [Alex-Rose/fb-messenger-cli#12]
I'm able to hit `/groups` from the main page, and then select a number of a group to enter. But once the messages are visible, trying to respond with _any_ text, and then hitting [enter], results in 

```
TypeError: Cannot read property 'vanity' of undefined
    at EventEmitter.Listeners.sendMessageListener (/usr/local/lib/node_modules/fb-messenger-cli/scripts/listeners.js:57:53)
```

**Comment by SamBergeron** [comment]
Looks like we broke the group functionality with the last modifications we made when facebook changed it's message API. Seems like not a lot of people use groups. I've got a fix coming tomorrow, just gotta make sure we don't break any more stuff with it! Thanks for the heads up.

**Comment by j6k4m8** [comment]
Let me know how I can be helpful!

**Comment by SamBergeron** [comment]
Should be fixed with version 1.2.3 Let me know if it holds up. You can update or reinstall with npm.
